.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
-exclude = .open-build-service/, abichecker, openqa, openqa-maintenance.py
+exclude = .open-build-service/, abichecker
```

Comment:
.open-build-service isn't here either. Basically all of 495878ff0f460b7f44dded36b8600ba8fdd8a794 can be reverted

Reply:
Done. Added a new commit for that.